src/components/EnrolledLearnersForInactiveCoursesTable/data/hooks/useCourseUsers.js

```diff
@@ -0,0 +1,95 @@
+import {
+  useCallback, useMemo, useRef, useState,
+} from 'react';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import debounce from 'lodash.debounce';
+import { logError } from '@edx/frontend-platform/logging';
+import EnterpriseDataApiService from '../../../../data/services/EnterpriseDataApiService';
+import EVENT_NAMES from '../../../../eventTracking';
+import { trackDataTableEvent } from '../../../LearnerActivityTable/data/utils';
+
+const applySortByToOptions = (sortBy, apiFieldsForColumnAccessor, options) => {
```

Comment:
This code is being used in two hooks, so I think we can move it to some common place by following the DRY pattern.